content/docs/user-guide/build/distributable-engine.md

```diff
@@ -0,0 +1,226 @@
+
+## Create a local SDK
+
+The first step in creating a distributable build is generating local binaries from your engine source, used to build your project and create the final distributable engine build.
+
+### Build the `INSTALL` target
+
+Perform the following steps from the O3DE source directory (`C:\o3de-source` in this example):
+
+1.  Generate the toolchain project files using a unique `LY_VERSION_ENGINE_NAME` CMake cache setting. This value is the name of the engine used by the O3DE project manager and registration system.
+    
+    ```cmd
+    cmake -B build/windows_vs2019 -G "Visual Studio 16" --config profile -DLY_VERSION_ENGINE_NAME="MyO3DE"
+    ```
+
+    The **profile** configuration is recommended for distributed builds, as it provides additional logging and engine introspection capabilities useful during project development at minimal performance cost. **debug** builds are primarily useful during engine development.
+1.  Build the `INSTALL` target.
+    
+    ```cmd
+    cmake --build build/windows_vs2019 --target INSTALL --config profile
+    ```
+    
+    The binaries will be placed into the directory specified by the `CMAKE_INSTALL_PREFIX` CMake cache variable, or in an `install` subdirectory of the source code by default.
+
+Run the following steps from the install directory:
+
+1. Install the version of Python and modules required by the engine.
+   
+   ```cmd
+   python\get_python.bat
+   ```
+
+### Register the engine
+
+The next step in creating a distributable build of the engine tools is to register the locally installed engine. You must re-run this registration step any time you make a change to `LY_VERSION_ENGINE_NAME`.
+
+Register the engine locally with the `o3de` script. Run the following command from your distributable build directory:
+    
+```cmd
+scripts\o3de.bat register --this-engine
+```
+
+{{< note >}}
+If there is a conflict in the existing `o3de_manifest.json` file on the system, the engine registration will fail and the `o3de` output provides information on how to fix the error. We recommend either automating this process through a script (for teams with rapid iteration requiring new builds) or using unique engine names for each new distributable (for teams requiring stability, rollback to previous builds outside of source control, or with infrequent distributable updates).
+{{< /note >}}
+
+At this point, the installed engine build is now registered in the O3DE config directory (by default `%USERPROFILE%\.o3de`) in the `o3de_manifest.json` file.
+
+## Project association
+
+At this point, you're ready to either create a new project for distribution or re-associate an existing project with the install build that will be used to create the distributable.
+
+### Create a new project
+
+Run the following command from the local engine source directory:
```

Comment:
Wait, shouldn't the new project be created from the distributable build directory, not the engine source directory?

Reply:
It should be created from the distributable install directory.

Using the engine source directory would result in the same project being created, BUT with the project being registered using the source engine name and not the SDK layout engine name which was set via the `LY_ENGINE_VERSION_NAME`

Reply:
Yes! That's exactly what I was worried about. Thanks for verifying. Will update.